### DISCUSSION TITLE
НЕ могу скачать!

### DISCUSSION THREAD

**Discussion by PoPik2** [community/community#141554] [Other Feature Feedback, Questions, & Ideas] [answered]
![image](https://github.com/user-attachments/assets/29788929-a50e-4ee8-b086-1b7166a99df3)

**Comment by queenofcorgis** [comment, marked as answer]
Hi @PoPik2 , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.

You are more likely to get a useful response if you are posting your question in the applicable category and include helpful details like context. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.
